docs/index.html

```diff
@@ -2142,6 +2142,15 @@ <h3 id="bundlemeta"><code>bundle.meta</code></h3>
 </div><div class="row">
   <div class="row-height">
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
+      <p><strong><code>bundle.meta.zap.id</code> is only available in the <code>performSubscribe</code> and <code>performUnsubscribe</code> methods</strong></p><p>The user&apos;s Zap ID is available during the <a href="https://github.com/zapier/zapier-platform-schema/blob/master/docs/build/schema.md#basichookoperationschema">subscribe and unsubscribe</a> methods.</p>
+    </div>
+    <div class="col-md-7 col-sm-12 col-height is-empty docs-code">
```

Comment:
Not sure how this got added

Reply:
I think the pattern here is a code bit for every explanation, since there's no code, this gets empty, but the page shouldn't be broken.